config/v3_2_experimental/schema/ignition.json

```diff
@@ -232,6 +238,70 @@
               "devices"
           ]
         },
+        "luks": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "label": {
+              "type": ["string", "null"]
+            },
+            "uuid": {
+              "type": ["string", "null"]
+            },
+            "device": {
+              "type": "string"
```

Comment:
Should be `["string", "null"]`.  Merged configs should be able to override properties without re-specifying `device`.